Solutions/Commvault Security IQ/SolutionMetadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "publisherId": "CommvaultSecurityIQ",
```

Comment:
This needs to be in lowercase.

Reply:
Made the changes and pushed it to the PR, all the validations are passing now, Thanks!